storage/rocksdb/ha_rocksdb.cc

```diff
@@ -19020,6 +19028,22 @@ static bool parse_fault_injection_params(
   return false;
 }
 
+static bool rocksdb_ddse_dict_init(
+    dict_init_mode_t dict_init_mode, [[maybe_unused]] uint version,
+    [[maybe_unused]] List<const dd::Object_table> *tables,
+    List<const Plugin_tablespace> *tablespaces) {
+  assert(tables && tables->is_empty());
+  assert(tablespaces && tablespaces->is_empty());
+
+  assert(dict_init_mode == DICT_INIT_CREATE_FILES ||
+         dict_init_mode == DICT_INIT_CHECK_FILES);
+
+  static Plugin_tablespace dd_space("mysql", "", "", "", rocksdb_hton_name);
+  tablespaces->push_back(&dd_space);
+
+  return false;
```

Comment:
We don't want this code to actually run, consider adding `assert(false)` and returning an error